src/com/owncloud/android/datamodel/OCFile.java

```diff
@@ -565,14 +565,16 @@ public long getLocalModificationTimestamp() {
      * @return 'True' if the file contains audio
      */
     public boolean isAudio() {
-        return (mMimeType != null && mMimeType.startsWith("audio/"));
+        return ((mMimeType != null && mMimeType.startsWith("audio/")) ||
+            getMimeTypeFromName().startsWith("audio/"));
```

Comment:
no isAudio, isVideo, isText, is Image all have the exact same code except for the string they check, so the code could be simplified into one generic, private method that is called by the for public ones.